openproblems/tasks/chromatin_potential/methods/linear_regression.py

```diff
@@ -0,0 +1,155 @@
+import numpy as np
+import pandas as pd
+from scipy.sparse import csr_matrix
+import scanpy as sc
+import os
+from ....tools.decorators import method
+
+
+def _chrom_limit(x):
+    y = x.values
+    if y[-1] == "+":
+        return [y[-3] - 1e5, y[-3] + 1e5]
```

Comment:
Where do these magic numbers come from?

Reply:
These number are from the pyensembl annotation of the gene body coordinates.

Reply:
this should be made much clearer. e.g.
```
def _chrom_limit(x, tss_size=2e5):
    y = x.values
    gene_direction = y[-1]
    gene_start = y[-3]
    gene_end = y[-2]
    if gene_direction  == "+":
        return [gene_start - tss_size // 2, gene_start + tss_size // 2]
    else:
        return [gene_end - tss_size // 2, gene_end + tss_size // 2]
```

Reply:
Thanks, I replaced it with your function.